Sources/KombuchaCLI/main.swift

```diff
@@ -27,7 +27,11 @@ sessionConfiguration.requestCachePolicy = .reloadIgnoringLocalCacheData
 sessionConfiguration = URLSessionConfiguration.ephemeral
 #endif
 
-sessionConfiguration.httpAdditionalHeaders = ["Accept": "application/json", "User-Agent": configuration.userAgent]
+let httpAdditionalHeaders = ["Accept": "application/json"]
```

Comment:
Should we be always forcing `"Accept": "application/json"` into the request? Will there ever be (and do we want to support) a case where someone will want to capture non-JSON information back from the server and use Kombucha to test it?

Reply:
for now only json :)